support-models/src/main/scala/com/gu/support/zuora/api/response/Responses.scala

```diff
@@ -200,3 +200,9 @@ object PreviewSubscribeResponse {
 }
 
 case class PreviewSubscribeResponse(invoiceData: List[InvoiceDataItem], success: Boolean) extends ZuoraResponse
+
+object UpdateRedemptionDataResponse {
+  implicit val codec: Codec[UpdateRedemptionDataResponse] = deriveCodec
```

Comment:
if its a response it would be a decoder not a codec I think

Reply:
A codec has both an encoder and decoder in it. It's convenient to have both in case you want to use an encoder in a test for instance.

Reply:
sure I would generally say YAGNI and add it (in the test code) if you do later. but not a major point.